hashing/te-tag-query-python/TE.py

```diff
@@ -159,9 +161,9 @@ def processDescriptorIDsByTagID(self, tagID, idProcessorCallback, **kwargs):
       "&limit=" + str(pageSize)
 
     if taggedSince != None:
-      startURL += "&tagged_since=" + taggedSince
+      startURL += "&tagged_since=" + urllib.parse.quote(taggedSince)
     if taggedUntil != None:
-      startURL += "&tagged_until=" + taggedUntil
+      startURL += "&tagged_until=" + urllib.parse.quote(taggedUntil)
```

Comment:
nit: it might be worth a comment noting that if 'taggedUntil' is None it is treated as 'now' (if my understanding is correct that is)

Reply:
Thanks!